frontend_tests/casper_tests/06-settings.js

```diff
@@ -78,6 +79,14 @@ casper.then(function () {
             confirm_password: test_credentials.default_user.password
         });
         casper.click('input[name="change_settings"]');
+
+        casper.test.assertExists('.download_zuliprc', 'Zuliprc button visible');
+        casper.click('a.download_zuliprc');
+        casper.test.assertMatch(
```

Comment:
I am wondering if we need some sort of watiUntilVisible() call here as well.  If we click the button and then immediately look at the href attribute, it's possible that the browser will still be rendering it, and this could lead to flaky failures.